sympy/printing/aesaracode.py

```diff
@@ -238,8 +240,13 @@ def _print_Piecewise(self, expr, **kwargs):
         return aet.switch(p_cond, p_e, p_remaining)
 
     def _print_Rational(self, expr, **kwargs):
-        return aet.true_div(self._print(expr.p, **kwargs),
-                            self._print(expr.q, **kwargs))
+        # `true_divide` replaced `true_div` in Aesara 2.8.11 to match NumPy
+        if aesara_version >= ('2', '8', '11'):
+            true_divide = aet.true_divide
+        else:
+            true_divide = aet.true_div
```

Comment:
It would less fragile to use something like:
```python
# `true_divide` replaced `true_div` in Aesara 2.8.11 (released 2023) to match NumPy
# XXX: Remove this when not needed to support older versions.
true_divide = getattr(aet, 'true_divide', None)
if true_divide is None:
    true_divide = aet.true_div
```
We can put that at the top of this module and then also import it into the test code.

Reply:
Addressed in https://github.com/sympy/sympy/pull/24747/commits/11d1d857041d352e538ed0159ef62157b2cf99c0.